airbyte-cdk/java/airbyte-cdk/core/src/main/resources/version.properties

```diff
@@ -1 +1 @@
-version=0.4.6
+version=0.4.7
```

Comment:
Don't forget to test with `useLocalCdk = true` and bump the CDK dependency for source-postgres.